ratis-server/src/main/java/org/apache/ratis/server/storage/SnapshotManager.java

```diff
@@ -61,22 +60,23 @@ public void installSnapshot(StateMachine stateMachine,
     final long lastIncludedIndex = snapshotChunkRequest.getTermIndex().getIndex();
     final RaftStorageDirectory dir = storage.getStorageDir();
 
-    // create a unique temporary directory
-    final File tmpDir =  new File(dir.getTmpDir(), UUID.randomUUID().toString());
```

Comment:
when receiving a snapshot which does not fit in a single chunk, then multiple directories are creates, each having a chunk of the snapshot